contracts/interfaces/IXTokenType.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.8.10;
+
+/**
+ * @title IXTokenType
+ * @author ParallelFi
+ * @notice Defines the basic interface for an IXTokenType.
+ **/
+enum XTokenType {
+    NoneType, // unused
```

Comment:
why do we need `NoneType`?

Reply:
no special reason. common way to prevent uninitialize zero value.

Reply:
why our AssetType doesn't need it?

Reply:
I add this NoneType  with no special reason, so don't need to care about it.